tree/ntuple/v7/test/ntuple_merger.cxx

```diff
@@ -1,5 +1,70 @@
 #include "ntuple_test.hxx"
 
+namespace {
+
+// Reads an integer from a little-endian 4 byte buffer
+std::int32_t ReadRawInt(void *ptr)
```

Comment:
Don't we already have this in a more generic place?

Reply:
> We have it as internal functions in RMiniFile.

What is the advantage of duplicating it in both place?

Reply:
Reading a 4 byte little endian buffer seemed like a small and innocent function.  In `RMiniFile.cxx` we have the four classes `RUInt16BE`, `RUInt32BE`, `RInt32BE`, and `RUInt64BE` in an anonymous namespace that implement the endianess swap but slightly different, ensuring big endian representation.

Those classes make sense in the context of the mini file implementation.  If we take them out and make them available as general infrastructure classes, more work is needed, e.g. support for the `LE` counterparts.